src/plugins/breezy_desktop.c

```diff
@@ -162,8 +162,18 @@ char* get_shared_mem_file_path() {
     return shared_mem_file_path;
 }
 
+static int fd = -2;
```

Comment:
Let's put `-2` as a `#define` since it's repeated in a few places and has special meaning.